### PR TITLE
Add Basic app layout

### LINKED ISSUE #11: Basic Layout of Canvas, Tools and Properties
Before we start working on any thing we will need to have the simple paned with a grid called canvas. we will replace the canvas grid with what `cairo` library needs and the left and right grid will be empty

### PR DESCRIPTION
**Pull request by Abdallah-Moh**
## This PR implements
This will implement the basic app layout to compile switch to this branch then follow instructions in readme
- Left Sidebar
- Right Sidebar
- Main Canvas

## Known issues / Things to do
- Add content to tools bar
- Style things with css

## This PR fixes
- Fixes #11

### CODE REVIEW

File: src/Layouts/MainWindow.vala [hunk 1]
```diff
@@ -23,11 +23,46 @@
...
+        right_sidebar = new Layouts.RightSideBar (window);
+        properties_bar = new Layouts.PropertiesBar (window);
+        main_canvas = new Layouts.MainCanvas (window);
+        tools_bar = new Photostat.Layouts.ToolsBar (window);
+
+        main_pane = new Gtk.Paned (Gtk.Orientation.HORIZONTAL);
+        child_pane = new Gtk.Paned (Gtk.Orientation.HORIZONTAL);
+
+        left_sidebar = new Gtk.Grid ();
+        left_sidebar.get_style_context ().add_class ("sidebar");
```
Comment (Suzie97) [inline src/Layouts/MainWindow.vala:51]: I think we should use `Gtk.STYLE_CLASS_SIDEBAR` instead of our own CSS

**Review by Suzie97 — COMMENTED** [review]
Looks good to me. Lets use the Gtk.STYLE_CLASS_SIDEBAR constant for consistency with the system.

File: src/Layouts/MainWindow.vala (hunk 1 above)
Comment (Abdallah-Moh) [reply src/Layouts/MainWindow.vala:51]: The background color

Comment (Suzie97) [reply src/Layouts/MainWindow.vala:51]: Are you talking about the background color of the sidebar?? 

Nvm, I'm merging this. If we want to use the `Gtk.STYLE_CLASS_SIDEBAR` class, it should be a small fix, and should break the code.

**Pull request merged**